tests/framework/installer/ceph_installer.go

```diff
@@ -145,6 +146,13 @@ func (h *CephInstaller) CreateCephOperator() (err error) {
 		return errors.Errorf("Failed to create rook-operator pod: %v", err)
 	}
 
+	manifest := h.settings.readManifest("csi-operator.yaml")
+	manifest = strings.ReplaceAll(manifest, `namespace: rook-ceph`, `namespace: `+h.settings.OperatorNamespace)
+	_, err = h.k8shelper.KubectlWithStdin(manifest, createFromStdinArgs...)
```

Comment:
This is just temporary for testing with csi operator, to revert before merge? But this isn't enabling the csi operator, looks like it's only installing it in the tests, so I'm not sure actually its purpose.

Reply:
no we need this fix as multicluster integration test enables external mode and we need CRD of csi-operator otherwise CI will fail when it will try to create cephConnection and client profile CR.

Reply:
Similar to comment above, it should only be required when csi operator is enabled, but we haven't enabled csi operator in any integration tests yet, until we decide the approach to merge #14766.

Reply:
corrected this.